pyqtgraph/colormap.py

```diff
@@ -330,25 +398,107 @@ def mapToFloat(self, data):
         return self.map(data, mode=self.FLOAT)
 
     def getByIndex(self, idx):
-        """Retrieve palette QColor by index"""
+        """Retrieve QColor by the index of the stop it is assigned to."""
         return QtGui.QColor( *self.color[idx] )
 
     def getGradient(self, p1=None, p2=None):
-        """Return a QLinearGradient object spanning from QPoints p1 to p2."""
+        """
+        Returns a `QtGui.QLinearGradient` object corresponding to the ColorMap.
+        The span and orientiation is given by two points in plot coordinates.
+        
+        When no parameters are given for `p1` and `p2`, the gradient is mapped to the 
+        `y` coordinates 0 to 1, unless the color map is defined for a more limited range.
+        
+        Parameters
+        ----------
+        p1: QtCore.QpointF, default (0,0)
```

Comment:
super nit-picky, I know but should be `QtCore.QPointF`

Reply:
⇒ changed.